test/extended/machines/cluster.go

```diff
@@ -79,6 +79,10 @@ var _ = g.Describe("[sig-node] Managed cluster", func() {
 	})
 
 	g.It("should report ready nodes the entire duration of the test run [Late]", func() {
+		_, _, _, prometheusAvailable := prometheus.LocatePrometheus(oc)
+		if !prometheusAvailable {
+			g.Skip("No prometheus available")
```

Comment:
@rphillips @joelsmith for awareness

We can't rely on Prometheus getting installed in every cluster. @rphillips is there any other way of detecting whether all the nodes were ready?

Reply:
@rphillips is this test supposed to be gating? Or, is it safe to skip it when Prometheus is not installed?

Reply:
Swapped the prometheus check with apigroups belonging to the cluster monitoring operator.